### ISSUE TITLE
MacOS X: focus on the display view acquired only after clicking in it

### ISSUE THREAD

**Issue by pcaillou** [gama-platform/gama.old#546]
```
It appears that, in the latest revision, the keyboard and mouse focus is transferred
to the AWT subsystem (shared by Java2D and OpenGL) only after the user has clicked
in the view. Moving the mouse into it is not enough (which is the case on Windows).
We'd need to test this under Ubuntu.

This problem of focus is a bit complicated, though, as transferring the focus too quickly
to the AWT subsystem appears to create a bug under Windows. So maybe we have to think
about a solution that would be specific to MacOS X.

Issue 540 and Issue 543 have been merged into this issue. 
```

Original issue reported on code.google.com by `alexis.drogoul` on 2013-08-26 04:35:03
